README.md

```diff
@@ -1,43 +1,40 @@
+
+A demo instance is available at https://demo.loris.ca.
+
+This Readme covers installation of the LORIS <b>v20.*</b> release on <b>Ubuntu</b>.
+([CentOS Readme also available](https://github.com/aces/Loris/blob/master/README.CentOS6.md)).
+
+Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+
+#### Heroku
+
+You can try LORIS on Heroku before installing it on your system. The project management and clinical data management functions of LORIS are available for experimenting with. Imaging functionality is not yet available.
 
-<hr>
-NEW <b>⇾  Try LORIS on Heroku</b> before installing it on your system<br>
-Test out the project management and clinical data management side of LORIS (complete Imaging features not yet available)<br>
 Deploy and log in with username <i>admin</i> and the password that's set up during deployment via ClearDB.
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/aces/Loris/tree/master)
-<hr>
-
-| Branch | Status |
-| ------ | ------ |
-| master | [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris)
-| major  | [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=major)](https://travis-ci.org/aces/Loris)
-| minor  | [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=minor)](https://travis-ci.org/aces/Loris)
-| bugfix | [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=bugfix)](https://travis-ci.org/aces/Loris)
+<br>
 
-This Readme covers installation of the LORIS <b>v20.*</b> release on <b>Ubuntu</b>.
-([CentOS Readme also available](https://github.com/aces/Loris/blob/master/README.CentOS6.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information not included in this Readme. The [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) may also provide installation guidance not covered in the Wiki. 
+## Installation
 
-# Prerequisites for Installation
+#### System Requirements
 
- * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
+ * Linux (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache2
  * MySQL 5.7 
  * PHP <b>7.2</b>  
- * Package manager (for LINUX distributions)
- * Composer : should be run with --no-dev option
+ * A package manager, e.g. `apt`
```

Comment:
Apache 2.4
MySQL 5.7+ or MariaDB 10.3+
PHP 7.2 or higher

We had a discussion on that in https://github.com/aces/Loris/pull/3976